File: FR/java/src/com/arcsoft/AFR_FSDK_FACEINPUT.java
package com.arcsoft;

import java.util.Arrays;
import java.util.List;

import com.sun.jna.Structure;

public class AFR_FSDK_FACEINPUT extends Structure {
    
    public MRECT.ByValue rcFace;
    public int lOrient;
    
    @Override
    protected List getFieldOrder() { 
        return Arrays.asList(new String[] { 
             "rcFace", "lOrient"
        });
    }
}

File: FR/C#/AFR_FSDK_FACEINPUT.cs
using System;
using System.Runtime.InteropServices;

namespace arcsoft {
    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Ansi)]
    public struct AFR_FSDK_FACEINPUT {
        public MRECT rcFace;
        public int lOrient;
    }
}
